### PR TITLE
WIP: Add paramiko

### PR DESCRIPTION
**Pull request by jakirkham**
Closes https://github.com/conda-forge/staged-recipes/issues/1106

Generated with `conda skeleton pypi` and cleaned up afterwards.

### CODE REVIEW

**Comment by jjhelmus** [comment]
LGTM.  Anything you were planning on adjusting before removing `WIP`?

**Comment by jakirkham** [comment]
Yeah, this recipe seems ready.

Was hoping to package its dependencies too. A few more maintainers wouldn't hurt. 😉

Though if you want to get this in sooner, I have no problems with that.

File: recipes/paramiko/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - paramiko
+
+about:
+  home: https://github.com/paramiko/paramiko/
+  license: LGPL 2.1
+  summary: SSH2 protocol library
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```
Comment (jakirkham) [inline recipes/paramiko/meta.yaml]: @danielfrg, @koverholt, @kwilcox, @ocefpaf, @vamega, it seems all of you have packages that depend on `paramiko` that you are maintaining. Do any of you have an interest in helping me maintain `paramiko`? This should help make sure it is in good shape for you to use elsewhere.

Comment (vamega) [inline recipes/paramiko/meta.yaml]: So far I've been using the one included in the Anaconda distribution.
This looks simple enough that I'd be able to help keep this updated rather easily.

**Comment by jakirkham** [comment]
Closing in favor of @pmlandwehr's PR ( https://github.com/conda-forge/staged-recipes/pull/1344 ) so that we can start with an older needed version and upgrade after. Please move the discussion over there.

**Pull request closed without merging**